### ISSUE TITLE
import/named for `module.exports`?

### ISSUE THREAD

**Issue by cbrwizard** [benmosher/eslint-plugin-import#1145]
Hey, is there any way to enable the `import/named` rule for `module.exports`?

Eg:

```
//lib.js
const toExport = true

module.exports = {
  toExport,
}


//user.js
const {toEpxort} = require('./lib') // here I want it to throw an error

console.log(toEpxort)
```

**Comment by ljharb** [comment]
It should, i'd expect.

**Comment by cbrwizard** [comment]
I'd definitely love that, as we haven't migrated backend code to ES6 imports yet. Looks like it doesn't work right now. Tried with this setting:
```
"import/named": [2, { "commonjs": true }],
```

**Comment by vikr01** [comment]
Should a case like this be ignored?

```javascript
if(foo === bar) {
  module.exports.foo = bar;
}
```

I assume we should only check the top-level `module.exports`/`exports`.

**Comment by ljharb** [comment]
I’m not worried about the stranger cases. Code that does that should be refactored to use a single static module.exports assignment.

**Comment by polgfred** [comment]
Any idea when this will get merged in?

**Comment by ljharb** [comment]
@polgfred as soon as there's something to merge - this is an issue, not a PR. Please send a PR :-)

**Comment by polgfred** [comment]
Sorry, I was referring to the open PR (#1222).

**Comment by polgfred** [comment]
Actually #1226 (commonjs exports) is the relevant one for this issue.

**Comment by fabioelizandro** [comment]
Is there anything I can help with this feature? I want to put this rule in place on my projects and in my option this feature suits really well `eslint-plugin-import` project

**Comment by ljharb** [comment]
@fabioelizandro if you comment a link to an updated branch on #1226, then I can pull in the changes and we can get it moving.
